lib/ansible/plugins/connection/local.py

```diff
@@ -129,7 +140,7 @@ def exec_command(self, cmd, in_data=None, sudoable=True):
         display.debug("done communicating")
 
         display.debug("done with local.exec_command()")
-        return (p.returncode, stdout, stderr)
+        return (p.returncode, become_stdout + stdout, become_stderr + stderr)
```

Comment:
If become_stdout was successful is it still helpful to keep the become_stderr?
I was also wondering if these should be separated with a newline. if I set the correct password in the playbook but use --ask-become-pass to give the wrong password initially, the task adds to the same line of the initial become error. For example, using the script in the issue that redirects AAAAA to stderr in the linked issue, the result looks like: `"stderr": "[sudo via ansible, key=azmobcdefgyjgozxrlbfkmjfw] password:AAAAA\n",`